src/r_plane.c

```diff
@@ -449,7 +450,8 @@ static void DrawSkyDef(visplane_t *pl)
 
     if (sky->type == SkyType_WithForeground)
     {
-        colfunc = R_DrawSkyColumnMasked;
+        tranmap = W_CacheLumpName("SKYTRAN", PU_CACHE);
```

Comment:
He, what is this?

Reply:
This is the trick that allows to use `R_DrawTLColumn` for drawing foreground sky. Quote from the spec:
> A custom render path is not required for the foreground sky to render transparently. All that
is needed is a correctly cooked transparency map, which results in the backbuffer texel
being used when the texture texel is a 0 index. While this map is generated at compile time
for our implementation, a copy of its contents can be found as SKYTRAN.lmp in the bundled
resources.

Reply:
Maybe you could leave a comment about this? I'm not sure if I'll still remember this next year. :wink:

Reply:
Fixed in https://github.com/fabiangreffrath/woof/pull/1894/commits/80855cc3114e32d6a1d4f7772327e5265eab83a2